kore/src/Kore/Builtin/Map.hs

```diff
@@ -181,17 +181,16 @@ expectBuiltinMap
     -> TermLike variable  -- ^ Operand pattern
     -> MaybeT m (Builtin variable)
 expectBuiltinMap ctx _map =
-    do
-        case _map of
-            DV_ _ domain ->
-                case domain of
-                    Domain.BuiltinMap Domain.InternalMap { builtinMapChild } ->
-                        return builtinMapChild
-                    _ ->
-                        Builtin.verifierBug
-                        $ Text.unpack ctx ++ ": Domain value is not a map"
-            _ ->
-                empty
+    case _map of
```

Comment:
Should we rename `_map` here to something else? Don't arguments that start with an underscore have special meaning to the linter/warning engine?

Reply:
Done